docs/src/modules/multilayerqg.md

```diff
@@ -1,5 +1,85 @@
 # MultiLayerQG
 
+
+### From continuous to discrete
+
+Under the quasi-geostrophic approximation with stratified dynamics both the horizontal flow, ``u(x, y, z, t)``, 
+``v(x, y, z, t)``, and the buoyancy ``b(x, y, z, t)`` are obtained via a streamfunction ``\psi(x, y, z, t)``:
+
+```math
+\begin{aligned}
+  u &=   - \partial_y \psi , \\
+  v &=     \partial_x \psi , \\
+  b &= f_0 \partial_z \psi .
+\end{aligned}
+```
+
+The dynamics are determined from the quasi-geostrophic potential vorticity (QGPV)
+
+```math
+\begin{aligned}
+  q &= \partial_x v - \partial_y u + \partial_z \left( \frac{f_0}{N^2} b \right) , \\
+    &= (\partial_x^2 + \partial_y^2) \psi + \partial_z \left( \frac{f_0^2}{N^2} \partial_z \psi \right) .
+\end{aligned}
+```
+
+where ``N^2 = - \rho_0^{-1} g \, \mathrm{d} \rho / \mathrm{d}z`` is the square of the Brunt-Väisälä
+frequency for a Boussinesq fluid, where ``\rho_0`` is a reference density and ``g`` the gravitational
+constant.
+
+To go from the continuous (in ``z``) dynamics above to a discrete dynamics we can integrate all
+quantities over the height of each fluid layer.
+An alternative is to replace the vertical derivatives that appear both in the buoyancy definition and
+in the Brunt-Väisälä frequency with a discrete approximation.
```

Comment:
Can't we also make the QG approximation directly in Navier-Stokes written in buoyancy coordinates?

Reply:
I'm confused whether the "layered QG" refers to QG in buoyancy / isopycnal coordinates, or whether its merely the a more vanilla vertically-finite-volume discretization of 3D QG in z-coordinates.